fbw-common/src/systems/shared/src/arinc429.ts

```diff
@@ -154,14 +160,24 @@ export class Arinc429Register implements Arinc429WordData {
   /**
    * Returns the value when normal operation, the supplied default value otherwise.
    */
-  valueOr(defaultValue: number | undefined | null) {
+  valueOr(defaultValue: number | undefined | null): number {
     return this.isNormalOperation() ? this.value : defaultValue;
   }
 
+  getBitValue(bit: number): boolean {
+    return ((this.value >> (bit - 1)) & 1) !== 0;
+  }
+
+  getBitValueOr(bit: number, defaultValue: boolean | undefined | null): boolean {
+    return this.isNormalOperation() ? ((this.value >> (bit - 1)) & 1) !== 0 : defaultValue;
+  }
+
+  /** @deprecated use {@link getBitValue} */
   bitValue(bit: number): boolean {
     return ((this.value >> (bit - 1)) & 1) !== 0;
   }
 
+  /** @deprecated use {@link getBitValueOr} */
```

Comment:
What's the reason for this change? `bitValue` and `bitValueOr` match up with `value` and `valueOr`. These new functions you've introduced do not match up with the rest of the API.

Reply:
Initially I was trying to use Arinc429ConsumerSubjects on the PFD side and as the bitValueOr methods weren't being exposed in the interface I picked the changes from here https://github.com/flybywiresim/aircraft/pull/8883#discussion_r1760085768. In any case, I've changed it so it all uses btValue & bitValueOr & removed the get functions so its consistent across the board.